[2024-09-22T19:59:36.104+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-22T19:59:36.115+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: python_example.task1 manual__2024-09-22T19:59:35.239145+00:00 [queued]>
[2024-09-22T19:59:36.120+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: python_example.task1 manual__2024-09-22T19:59:35.239145+00:00 [queued]>
[2024-09-22T19:59:36.121+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-22T19:59:36.131+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): task1> on 2024-09-22 19:59:35.239145+00:00
[2024-09-22T19:59:36.135+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=153) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-22T19:59:36.136+0000] {standard_task_runner.py:72} INFO - Started process 155 to run task
[2024-09-22T19:59:36.137+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'python_example', 'task1', 'manual__2024-09-22T19:59:35.239145+00:00', '--job-id', '786', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpwptjw34f']
[2024-09-22T19:59:36.138+0000] {standard_task_runner.py:105} INFO - Job 786: Subtask task1
[2024-09-22T19:59:36.167+0000] {task_command.py:467} INFO - Running <TaskInstance: python_example.task1 manual__2024-09-22T19:59:35.239145+00:00 [running]> on host 677b929c6314
[2024-09-22T19:59:36.352+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='python_example' AIRFLOW_CTX_TASK_ID='task1' AIRFLOW_CTX_EXECUTION_DATE='2024-09-22T19:59:35.239145+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-22T19:59:35.239145+00:00'
[2024-09-22T19:59:36.352+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-22T19:59:36.363+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-22T19:59:36.364+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/***/script/generate_data.py /opt/***/input 2024-09-15 30 10 2000']
[2024-09-22T19:59:36.371+0000] {subprocess.py:86} INFO - Output:
[2024-09-22T19:59:36.719+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-22T19:59:36.736+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-22T19:59:36.736+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=python_example, task_id=task1, run_id=manual__2024-09-22T19:59:35.239145+00:00, execution_date=20240922T195935, start_date=20240922T195936, end_date=20240922T195936
[2024-09-22T19:59:36.751+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-22T19:59:36.764+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-22T19:59:36.765+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
